cogs5e/sheets/gsheet.py

```diff
@@ -198,7 +198,7 @@ def _gchar(self):
         vcell = self.character_data.value("AQ4")
         if ("1.3" not in vcell) and vcell:
             self.additional = TempCharacter(doc.get_worksheet(1))
-            self.version = 2 if "2" in vcell else 1
+            self.version = 2.1 if "2.1" in vcell else 2 if "2" in vcell else 1
```

Comment:
Generally, it's better to keep version numbers as a tuple of `(major, minor, patch)` (i.e. 2.1 should be version `(2, 1)`) - previously this didn't matter since there wasn't any difference in between minor versions

Reply:
I assume then that this would necessitate changing the various instances of `self.version >= 2` to something else then as well?

Reply:
Yup, Python handles this pretty easily:
```py
>>> (2, 1) >= (2, 0)
True
>>> (2, 0) >= (2, 1)
False
>>> (1, 0) >= (2, 0)
False
>>> (1, 4) >= (2, 0)
False
```